hmpps-kotlin-spring-boot-autoconfigure/src/main/kotlin/uk/gov/justice/hmpps/kotlin/auth/HmppsWebClientConfiguration.kt

```diff
@@ -24,9 +25,13 @@ import reactor.netty.http.client.HttpClient
 import java.time.Duration
 import kotlin.apply as kotlinApply
 
+private const val DEFAULT_TIMEOUT_SECONDS: Long = 30
+private const val DEFAULT_HEALTH_TIMEOUT_SECONDS: Long = 2
+
 @ConditionalOnWebApplication(type = SERVLET)
 @Configuration
 class HmppsWebClientConfiguration {
+  @ConditionalOnBean(ClientRegistrationRepository::class)
```

Comment:
If there are no client registrations configured then Spring doesn't create a `ClientRegistrationRepository` bean and this bean creation fails so the app won't start

Reply:
not sure it matters - but should this be at the `class` level instead - why put it at method level?